<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SortStudentWidget</class>
 <widget class="QWidget" name="SortStudentWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="returnButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>40</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>主菜单 </string>
   </property>
  </widget>
  <widget class="QRadioButton" name="mathRadioButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>数学分析</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="enRadioButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>英语四级</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="cRadioButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>57</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>C++</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="avgRadioButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>47</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>平均成绩</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sortButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>40</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>排序</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>591</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
